app/controllers/api/v1/external_resources_controller.rb

```diff
@@ -0,0 +1,312 @@
+module Api
+  module V1
+    class ExternalResourcesController < ApiBaseController
+
+      include Concerns::FireCloudStatus
+      include Concerns::Authenticator
+      include Swagger::Blocks
+
+      before_action :authenticate_api_user!
+      before_action :set_study
+      before_action :check_study_permission
+      before_action :set_external_resource, except: [:index, :create]
+
+      respond_to :json
+
+      swagger_path '/studies/{study_id}/external_resources' do
+        operation :get do
+          key :tags, [
+              'ExternalResources'
+          ]
+          key :summary, 'Find all ExternalResources in a Study'
+          key :description, 'Returns all ExternalResources for the given Study'
+          key :operationId, 'study_external_resources_path'
+          parameter do
+            key :name, :study_id
+            key :in, :path
+            key :description, 'ID of Study'
+            key :required, true
+            key :type, :string
+          end
+          response 200 do
+            key :description, 'Array of ExternalResource objects'
+            schema do
+              key :type, :array
+              key :title, 'Array'
+              items do
+                key :title, 'ExternalResource'
+                key :'$ref', :ExternalResource
+              end
+            end
+          end
+          response 401 do
+            key :description, ApiBaseController.unauthorized
+          end
+          response 403 do
+            key :description, ApiBaseController.forbidden('edit Study')
+          end
+          response 404 do
+            key :description, ApiBaseController.not_found(Study)
+          end
+          response 406 do
+            key :description, ApiBaseController.not_acceptable
+          end
+        end
+      end
+
+      # GET /single_cell/api/v1/studies/:study_id
+      def index
+        @external_resources = @study.external_resources
+      end
+
+      swagger_path '/studies/{study_id}/external_resources/{id}' do
+        operation :get do
+          key :tags, [
+              'ExternalResources'
+          ]
+          key :summary, 'Find a ExternalResource'
```

Comment:
```suggestion
          key :summary, 'Find an ExternalResource'
```

And similarly for other instances of "a ExternalResource".